source/_docs/managing-wordpress-site-networks.md

```diff
@@ -38,22 +38,26 @@ terminus env:clear-cache <site>.test
 Code that has been committed to master and is running on the Dev environment:
 ```bash
 terminus env:deploy <site>.test --sync-content --cc --updatedb --note=<note>
-terminus wp <site>.test -- search-replace $DOMAIN $TESTDOMAIN --url=$DOMAIN --network
+terminus remote:wp <site>.test -- search-replace $DOMAIN $TESTDOMAIN --url=$DOMAIN --network
 terminus env:clear-cache <site>.test
 ```
 ## Clone Content from Live to Test
 Restore the Test database and files to the state of the Live environment.
 ```bash
-env:clone <site>.live test
-terminus wp <site>.test -- search-replace $DOMAIN $TESTDOMAIN --url=$DOMAIN --network
+terminus env:clone <site>.live test
+terminus remote:wp <site>.test -- search-replace $DOMAIN $TESTDOMAIN --url=$DOMAIN --network
 terminus env:cc <site>.test
```

Comment:
The doc uses `env:cc` and `env:clear-cache`, let's standardize on `env:clear-cache`